frontend/src/components/dashboard/UploadsFields.js

```diff
@@ -30,29 +30,46 @@ export default function UploadsFields() {
         justify="flex-start"
         alignItems="center"
       >
-        <form className={classes.root} noValidate autoComplete="off">
+        <form 
+          className={classes.root} 
+          noValidate 
+          autoComplete="off"
+          action="/api/v1/uploadInfo"
+          method="POST"
+        >
           <div>
             <TextField
               required
-              id="first-name"
-              label="First Name"
+              id="artist-name"
+              label="Artist Name"
+              name="artistName"
               variant="outlined"
             />
           </div>
           <div>
-            <TextField id="last-name" label="Last Name" variant="outlined" />
+            <TextField 
```

Comment:
I would run npx eslint because Github Actions found a few spacing fixes (not sure if you can see them on your side but for instance, on line 50 it says there is a trailing space). If you run the npx command with --fix it should do it for you!